### PR TITLE
updates screenshots for temporary installation in firefox

### PR DESCRIPTION
**Pull request by caitmuenster**
This should change the video screenshots in the temporary installation in Firefox page.

### CODE REVIEW

**Comment by caitmuenster** [comment]
@muffinresearch Can you check this out?

File: documentation/develop/temporary-installation-in-firefox.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ topic: Develop
 tags: [WebExtensions]
 contributors:
   [
+    caitmuenster
```
Comment (muffinresearch) [inline documentation/develop/temporary-installation-in-firefox.md]: This line is missing a comma at the end of it.

**Comment by muffinresearch** [comment]
I fixed the missing comma locally, it looks like this:

Screen Shot 2019-12-19 at 17 56 18-fullpage

I wonder if there's a way to avoid the you-tube icons since they looks slightly at odds with the play button?

That said if you want to ship this and fix that later I'm ok with that. If so the only chance needed is to fix the missing comma.

**Comment by muffinresearch** [comment]
I still can't get this to build in time for circle not to time out. 

One nit, is the newer images without the youtube logo are in the wrong location.

**Comment by muffinresearch** [comment]
> I still can't get this to build in time for circle not to time out. I'm running some additional profiling currently to try and pin down where the slowness is.
> 
> Failing getting somewhere with this we might just need to bite the bullet and switch to an alternative.

This part should be fixed by https://github.com/mozilla/extension-workshop/pull/544 which is landed, so if you pull the recent changes this should now build without getting killed 🤞.

> 
> Note: One nit in the PR, is the newer images without the youtube logo are in the wrong location.

Once the new images replace the old ones this should be good to go.

**Comment by caitmuenster** [comment]
@muffinresearch I have to admit defeat on this -- I can't figure out how to move the image file to the correct folder and use it in the video pop-up module: 

Screen Shot 2020-01-08 at 10 41 23 AM

The clean versions of the images are at: 
* [Temporary installation in Firefox ](https://drive.google.com/a/mozilla.com/file/d/1DRzj6QbdJN9-7ofeKe4nlNRjjEpFkGkf/view?usp=sharing)
* [Reload temporary extensions](https://drive.google.com/a/mozilla.com/file/d/1ZeIWpBZ-DlNipDtfPxggEfuQHDf5ixBc/view?usp=sharing) 

Could someone else put the images in the right place and do the replacement?

**Comment by muffinresearch** [comment]
> Could someone else put the images in the right place and do the replacement?

@caitmuenster yep I'll take a look.

**Comment by muffinresearch** [comment]
I've moved the images and re-built everything and it looks like this:

Screen Shot 2020-01-09 at 18 54 28-fullpage

**Comment by muffinresearch** [comment]
@caitmuenster once this build is green and circle-ci is happy, feel free to go ahead and land this to check it on -dev.

**Comment by caitmuenster** [comment]
Thank you @muffinresearch! ❤️

**Pull request merged**